yldio/asap-hub
# CRN-969 | Backend work to enable users to dismiss "Getting Started" dialog.

# CRN-969 | Backend work to enable users to dismiss "Getting Started" dialog

## JIRA Ticket

[CRN-969](https://asaphub.atlassian.net/jira/software/c/projects/CRN/boards/8?modal=detail&selectedIssue=CRN-969)

## Description of Changes

- Adds the ability to set and retrieve `dismissedGettingStarted` toggle in Squidex
- Updates the Squidex schema to add the boolean property
- Removes the feature flag
- Updates the subtext

## Testing Instructions

These tests can be performed on the [PR Branch](https://1986.hub.asap.science/).

1. Log in
2. On the home page, you will see the 'Getting Started with ASAP' section
3. Click the button labelled 'Don't Show Again' and confirm on the modal
4. You should see the section disappear, and the modal should dismiss itself
5. The subtext of the header should also be removed

If you need to re-perform the test, you will need to log into Squidex and toggle `dismissedGettingStarted` off for your
